odoo/tools/translate.py

```diff
@@ -1580,6 +1580,21 @@ def _get_translation_upgrade_queries(cr, field):
         cleanup_queries.append(cr.mogrify(query, [translation_name]).decode())
 
     # upgrade model_terms translation: one update per field per record
+    query = f"""
+        SELECT t.res_id, m."{field.name}", t.value, t.noupdate
+          FROM t
+          JOIN "{Model._table}" m ON t.res_id = m.id
+    """
+    cr.execute("SELECT id from ir_module_module WHERE name = 'website' AND state='installed'")
```

Comment:
This extra query is only necessary for field `ir.ui.view.arch_db`, you don't need it for other fields